src/App.tsx

```diff
@@ -47,6 +47,13 @@ function App() {
       setCheckedCoinBottom(evt);
     }
   };
+
+  const changeToken = (evt: any) => {
+    evt.stopPropagation();
+    setCheckedCoinTop(checkedCoinBottom);
+    setCheckedCoinBottom(checkedCoinTop);
```

Comment:
also when switching, also the amount the user entered in the input should follow the "switch"

Reply:
fixed this.